_static/devguide_overrides.css

```diff
@@ -7,66 +7,70 @@
 }
 
 /* Release cycle chart */
-#python-release-cycle .mermaid .active0,
-#python-release-cycle .mermaid .active1,
-#python-release-cycle .mermaid .active2,
-#python-release-cycle .mermaid .active3 {
-    fill: #00dd00;
-    stroke: darkgreen;
+
+.release-cycle-chart {
+    width: 100%;
+    /* filter: grayscale(100%); */
+}
+
+.release-cycle-chart .release-cycle-year-line {
+    stroke: var(--color-foreground-primary);
+    stroke-width: 0.8px;
+    opacity: 75%;
+}
+
+.release-cycle-chart .release-cycle-year-text {
+    fill: var(--color-foreground-primary);
+}
+
+.release-cycle-chart .release-cycle-today-line {
+    stroke: var(--color-brand-primary);
+    stroke-width: 1.6px;
 }
 
-#python-release-cycle .mermaid .done0,
-#python-release-cycle .mermaid .done1,
-#python-release-cycle .mermaid .done2,
-#python-release-cycle .mermaid .done3 {
-    fill: orange;
-    stroke: darkorange;
+.release-cycle-chart .release-cycle-row-shade {
+    fill: var(--color-background-item);
+    opacity: 50%;
 }
 
-#python-release-cycle .mermaid .task0,
-#python-release-cycle .mermaid .task1,
-#python-release-cycle .mermaid .task2,
-#python-release-cycle .mermaid .task3 {
-    fill: #007acc;
-    stroke: #004455;
+.release-cycle-chart .release-cycle-version-label {
+    fill: var(--color-foreground-primary);
 }
 
-#python-release-cycle .mermaid .section0,
-#python-release-cycle .mermaid .section2 {
-    fill: darkgrey;
+.release-cycle-chart .release-cycle-blob {
+    stroke-width: 1.6px;
+    /* default colours, overriden below for individual statuses */
+    fill: var(--color-background-primary);
+    stroke: var(--color-foreground-primary);
 }
 
-/* Set master colours */
-:root {
-    --mermaid-section1-3: white;
-    --mermaid-text-color: black;
+.release-cycle-chart .release-cycle-blob-label {
+    /* white looks good on both light & dark */
+    fill: white;
+    filter:
+        drop-shadow(1px 1px 0.5px rgba(0, 0, 0, 0.5))
+        drop-shadow(-1px 1px 0.5px rgba(0, 0, 0, 0.5))
+        drop-shadow(1px -1px 0.5px rgba(0, 0, 0, 0.5))
+        drop-shadow(-1px -1px 0.5px rgba(0, 0, 0, 0.5))
+        ;
```

Comment:
Maybe drop the drop shadow and add a contrasting stroke instead? That might resolve the text readability issue I'm having

Reply:
Updated, how's this?

# Before

<img width="768" alt="image" src="https://user-images.githubusercontent.com/1324225/213916108-a3cdbf57-4585-428c-8553-181f75fe98f1.png">

# After

<img width="763" alt="image" src="https://user-images.githubusercontent.com/1324225/213916125-3b3d61a0-5dca-4b50-b212-39e594821628.png">

Reply:
LGTM!